packages/protocol/test/common/metatransactionwallet.ts

```diff
@@ -304,7 +304,15 @@ contract('MetaTransactionWallet', (accounts: string[]) => {
             await wallet.executeTransactions(
               transactions.map((t) => t.destination),
```

Comment:
attempted to DRY up this parameter generation, but I clearly need to get up to speed on Typescript...